pkg/utils/nodeutils.go

```diff
@@ -213,11 +215,19 @@ func IsKCSubsetOfMC(kc *mcfgv1.KubeletConfig, mc *mcfgv1.MachineConfig) (bool, e
 		return false, fmt.Errorf("encoded kubeletconfig %s does not contain encoding prefix", mc.Name), ""
 	}
 
+	// remove node sizing related fields from kubelet config
+	filteredKC, err := removeNodeSizingEnvParams(kc.Spec.KubeletConfig.Raw)
+	if err != nil {
+		return false, fmt.Errorf("failed to remove node sizing related fields from decoded kubelet config: %w", err), ""
+	}
+	fmt.Println(string(filteredKC))
```

Comment:
is this a leftover debug string or did you mean to leave the println in?

Reply:
oops, forgot to remove that.